projects/1.20/assets/cybernetic-system/cybernetic_system/lang/zh_cn.json

```
@@ -0,0 +1,899 @@
+  "entity.cybernetic_system.mb_1": "MB1",
+  "entity.cybernetic_system.mb_2_hellhound": "MB2 地狱猎犬",
+  "entity.cybernetic_system.mdg_1": "MDG1",
+  "entity.cybernetic_system.tm_1": "TM-1",
+  "gui.cybernetic_system.gui_ability_cyborg.label_10_xp": "10 XP",
+  "gui.cybernetic_system.gui_ability_cyborg.label_8_xp": "8 XP",
+  "gui.cybernetic_system.gui_ability_cyborg.label_8_xp1": "8 XP",
+  "gui.cybernetic_system.gui_ability_cyborg.label_8_xp2": "8 XP",
+  "gui.cybernetic_system.gui_ability_cyborg.label_8_xp3": "8 XP",
+  "gui.cybernetic_system.gui_ability_cyborg.tooltip_allows_you_to_use_abilities_when": "能量高于30%时可使用技能。",
+  "gui.cybernetic_system.gui_ability_cyborg.tooltip_body_booster": "躯体增幅",
+  "gui.cybernetic_system.gui_ability_cyborg.tooltip_electric_field": "电场",
+  "gui.cybernetic_system.gui_ability_cyborg.tooltip_energy_attack": "能量攻击",
+  "gui.cybernetic_system.gui_ability_cyborg.tooltip_healing": "治疗",
+  "gui.cybernetic_system.gui_ability_cyborg.tooltip_teleport": "闪现",
+  "gui.cybernetic_system.gui_atomic_heart.label_20": "20",
+  "gui.cybernetic_system.gui_atomic_heart.label_32": "32",
+  "gui.cybernetic_system.gui_atomic_heart.label_5": "5",
+  "gui.cybernetic_system.gui_atomic_heart.label_51": "5",
+  "gui.cybernetic_system.gui_atomic_heart.label_is_required": "需要：",
+  "gui.cybernetic_system.gui_atomic_heart.tooltip_create": "制造",
+  "gui.cybernetic_system.gui_base_station_two.label_lost_blueprints": "遗失蓝图",
+  "gui.cybernetic_system.gui_base_station_two.tooltip_boots_lg4f": "LG4F靴子",
+  "gui.cybernetic_system.gui_base_station_two.tooltip_boots_lg6p": "LG6P靴子",
+  "gui.cybernetic_system.gui_base_station_two.tooltip_chestplate_lg4f": "LG4F胸甲",
+  "gui.cybernetic_system.gui_base_station_two.tooltip_chestplate_lg6p": "LG6P胸甲",
+  "gui.cybernetic_system.gui_base_station_two.tooltip_gc2b": "GC2B",
+  "gui.cybernetic_system.gui_base_station_two.tooltip_helmet_lg4f": "LG4F头盔",
+  "gui.cybernetic_system.gui_base_station_two.tooltip_helmet_lg6p": "LG6P头盔",
+  "gui.cybernetic_system.gui_base_station_two.tooltip_lc2_boots": "LC2靴子",
+  "gui.cybernetic_system.gui_base_station_two.tooltip_lc2_chestplate": "LC2胸甲",
+  "gui.cybernetic_system.gui_base_station_two.tooltip_lc2_helmet": "LC2头盔",
+  "gui.cybernetic_system.gui_base_station_two.tooltip_lc2_leggings": "LC2护腿",
+  "gui.cybernetic_system.gui_base_station_two.tooltip_leggings_lg4f": "LG4F护腿",
+  "gui.cybernetic_system.gui_base_station_two.tooltip_leggings_lg6p": "LG6P护腿",
+  "gui.cybernetic_system.gui_base_station.label_other_resources": "其他材料",
+  "gui.cybernetic_system.gui_base_station.label_weaponsarmor": "武器/盔甲",
+  "gui.cybernetic_system.gui_base_station.tooltip_armor_l1_chestplate": "L1装甲- 胸甲",
+  "gui.cybernetic_system.gui_base_station.tooltip_armor_l1_helmet": "L1装甲- 头盔",
+  "gui.cybernetic_system.gui_base_station.tooltip_armor_l1boots": "装甲 L1-靴子",
+  "gui.cybernetic_system.gui_base_station.tooltip_armor_l1leggings": "L1装甲-护腿",
```

```suggestion
  "gui.cybernetic_system.gui_base_station.tooltip_armor_l1_chestplate": "L1装甲 - 胸甲",
  "gui.cybernetic_system.gui_base_station.tooltip_armor_l1_helmet": "L1装甲 - 头盔",
  "gui.cybernetic_system.gui_base_station.tooltip_armor_l1boots": "L1装甲 - 靴子",
  "gui.cybernetic_system.gui_base_station.tooltip_armor_l1leggings": "L1装甲 - 护腿",
```
连接号前后空格随意，建议按照enus，但需各处统一
此处仅作示例，真正需要修改的部分是第三行的L1装甲

---

我会检查所有的空格保持格式一致

---

已更改